README.md

```diff
@@ -20,7 +20,7 @@ Please note that, In order to use this pacakge, `@tizentv/webIDE-common-tizentv`
 
 ```js
     constructor(resourcePath) ;
-
+    init();
     /*
     authorInfo {
```

Comment:
In my opinion....
"author" is too redundant.

I think the name of previous parameters looks ok. What do you think?
const authorInfo = {
  keyfileName: '', 
  authorName: '',
  authorPassword: '',
  countryInfo: '',
  stateInfo: '',
  cityInfo: '',
  organizationInfo: '',
  departmentInfo: '',
  emailInfo: ''
}

Reply:
done